orca/internal/internal.go

```diff
@@ -20,7 +20,16 @@
 // avoid polluting the godoc of the top-level orca package.
 package internal
 
-import ibackoff "google.golang.org/grpc/internal/backoff"
+import (
+	"errors"
+	"fmt"
+
+	ibackoff "google.golang.org/grpc/internal/backoff"
```

Comment:
I think I saw Easwar comment something about import renaming with no alias/conflict and you replied with to keep it consistent with other packages that conflict with backoff. Does this apply to this?

Reply:
`backoff` is a top-level package so I wouldn't want to use that name.  It looks like we use the more verbose "internalbackoff" in other places. This usage is existing so I'd rather just leave it alone and avoid churn.

Reply:
Ok. I always get confused with balancer/ vs. internal/balancer conflicts. Maybe we should start adopting conventions for internal packages prefixed with i.

Reply:
I'd be fine with something like that...having consistent names is always good.